server/apis/v1/handler.go

```diff
@@ -824,6 +831,19 @@ func getPipelineStatus(pipeline *dfv1.Pipeline) (string, error) {
 	return retStatus, nil
 }
 
+// GetIsbServiceStatus is used to provide the status of a given InterStepBufferService
+func getIsbServiceStatus(isbsvc *dfv1.InterStepBufferService) (string, error) {
+	retStatus := ISBServiceStatusHealthy
```

Comment:
We will figure out the correct way of determining if pipeline/isbsvc/vertex is healthy later on, can you please add a TODO for that? But this is good to clean up those TODOs to abstract to this function.